pkgs/applications/networking/cluster/kfilt/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, buildGo120Module, fetchFromGitHub }:
+
+buildGo120Module rec {
```

```suggestion
buildGoModule rec {
```

`buildGoModule` already points to `1.20` and should be preferred.

If this package happens to fail to build with some new go version `buildGoModule` then points to, another maintainer temporarily overwrites this in the `all-packages.nix` instead :)